### ISSUE TITLE
[1.x] Statistics don't get logged

### ISSUE THREAD

**Issue by amcsi** [beyondcode/laravel-websockets#719]
Version: `1.11.1`.

The websocket/pusher functionality works just fine... but no matter if in the local environment or on our dev environment, no matter if using SQL Server or MySQL, _statistics_ do not get logged.
The migrations are there, they've been run, and every setting is on its default. In fact, the project is just a fresh Laravel project with laravel-websockets installed.

I know that in `\BeyondCode\LaravelWebSockets\Statistics\Logger\HttpStatisticsLogger::save()` we do get to this part:

```php
            $this
                ->browser
                ->post(
                    action([WebSocketStatisticsEntriesController::class, 'store']),
                    ['Content-Type' => 'application/json'],
                    stream_for(json_encode($postData))
                );
``` 

But then it all gets swallowed up there. I put a `\Log::warning('here');` at the top of `\BeyondCode\LaravelWebSockets\Statistics\Http\Controllers\WebSocketStatisticsEntriesController::store()`, but it does not get called. There's no other exception/message logged anywhere.

I don't really understand this "browser" thing of how it works to debug it.

**Comment by rodrigoross** [comment]
Well I had the same issue. But after 1 day trying to fix I just gave up and updated to laravel-websockets 2.x. 

It started working and now I'm using 2.0.0-beta.33. And don't forget to copy the new websocket.php to your config/websocket.php. Somehow even after I upgraded it still wasn't logging in the dashboard, but at least I managed to workaround.

Here my [fix](https://github.com/beyondcode/laravel-websockets/issues/716) after upgrading to 2.x.
